docs/admin/addons.md

```diff
@@ -14,6 +14,8 @@ This page lists some of the available add-ons and links to their respective inst
 * [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm) unites Flannel and Calico, providing networking and network policy.
 * [Romana](http://romana.io) is a Layer 3 networking solution for pod networks that also supports the [NetworkPolicy API](/docs/user-guide/networkpolicies/). Kubeadm add-on installation details available [here](https://github.com/romana/romana/tree/master/containerize).
 
+Limitation: Due to [#31307](https://github.com/kubernetes/kubernetes/issues/31307), `HostPort` won't work with CNI networking plugin at the moment. That means all `hostPort` attribute in pod would be simply ignored.
```

Comment:
Not sure this is the right place for this note since this isn't really talking about CNI specifically and should just be a list of network plugins?

Maybe it belongs here instead http://kubernetes.io/docs/admin/network-plugins/#cni

WDYT?